firefox-ios/Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -636,6 +636,7 @@ extension TelemetryWrapper {
         case crashedLastLaunch = "crashed_last_launch"
         case cpuException = "cpu_exception"
         case hangException = "hang-exception"
+        case tabLossDetected = "tab-loss-detected"
```

Comment:
@adudenamedruby @OrlaM Fyi, I just noticed some of our events (e.g. `hang-exception`) use dashes for the enum string itself but underscores (`hang_exception`) in the metrics.yaml. I followed this pattern, but I think it deviates from the other errors, which appear to use underscores everywhere (e.g. `cpu_exception`).

Not sure if any of this matters? But if so LMK and I can fix if needed.

Reply:
Good catch, underscores seem to be the most common in this file so I would stick with underscores, "hang-exception" is the mistake.